### PR TITLE
Merge of SLE-12-SP2_CASP into master

### PR DESCRIPTION
**Pull request by mchf**

### CODE REVIEW

File: package/yast2-installation.changes [hunk 1]
```diff
@@ -1,4 +1,14 @@
 -------------------------------------------------------------------
+Thu Mar  6 13:33:41 UTC 2017 - mfilka@suse.com
+
+- fate#322328
+  - adapted all-in-one dialog according to comments from UX
+- bnc#1022784
+  - Downloading release notes: Do not try again if the host or the
+    proxy cannot be resolved (schubi@suse.de)
```
Comment (mvidner) [inline package/yast2-installation.changes]: No, this is already mentioned at 3.2.19 below. The fact that it doesn't show up in the diff should have warned you.

File: package/yast2-installation.changes [hunk 2]
```diff
@@ -1,4 +1,14 @@
...
+- 3.1.25
```
Comment (mvidner) [inline package/yast2-installation.changes]: 3.2.25

File: package/yast2-installation.changes (hunk 1 above)
Comment (mchf) [inline package/yast2-installation.changes]: ah sorry

Comment (mchf) [inline package/yast2-installation.changes]: local diff when merging confused me a bit ;-)

**Comment by mchf** [comment]
thanks for the review

**Pull request merged**